front/src/common/Map/Layers/MastsLayers.tsx

```diff
@@ -0,0 +1,52 @@
+/**
+ * Shared layer showing the mast for icons on the map
+ */
+import { SymbolLayer } from 'react-map-gl/maplibre';
+import { isNil } from 'lodash';
+import { OmitLayer } from 'types';
+
+/**
+ * Shared layer showing the mast for icons on the map
+ * @param params
+ */
+export function getCommonSignsMastLayerProps(params: {
+  anglePropertyName?: 'angle' | 'angle_geo';
+  sourceTable: string;
+  minzoom?: number;
+  maxzoom?: number;
+}): OmitLayer<SymbolLayer> {
```

Comment:
I'm not sure all these params are needed: if I'm not wrong, you never give the minzoom or maxzoom, so we can remove them, and you never give the anglePropertyName, so we can juste remove it since it will always be 'angle' by default :)
Moreover, it's better to keep the typing from the parent type LayerContext:
  
```suggestion
export function getCommonSignsMastLayerProps({
  sourceTable,
}: Pick<LayerContext, 'sourceTable'> {
```

Reply:
corrected